eng/common/pipelines/templates/steps/publish-blobs.yml

```diff
@@ -25,6 +25,8 @@ steps:
       -BlobName "${{ parameters.BlobName }}"
       -PublicArtifactLocation "${{ parameters.ArtifactLocation }}"
       -RepoReplaceRegex "(https://github.com/${{ parameters.RepoId }}/(?:blob|tree)/)master"
+      -WorkingDirectory "$(System.DefaultWorkingDirectory)"
+      -ReleaseSha: "$(Build.SourceVersion)"
```

Comment:
This doesn't look correct. Why are we passing the SourceVersion as the sha?

Reply:
From my understanding the $(Build.SourceVersion) will be the SHA for the current release. Is there a situation where that will not be the case?

Reply:
No, releaseTag is not SHA commit. It is like "azure-template_1.0.0-beta.1"

Reply:
Sorry, I thought it was releaseTag variable. $releaseSha is SHA commit to use in `VerifyPackages` method

Reply:
Based on the update we should be able to undo this change